docs/ibc/overview.md

```diff
@@ -6,170 +6,186 @@ parent:
+- In UNORDERED channels, packets can be received in any order. Thus, IBC writes a packet receipt for each sequence it has received in the UNORDERED channel. This receipt contains no information; it is simply a marker intended to signify that the UNORDERED channel has received a packet at the specified sequence. To timeout a packet on an UNORDERED channel, one must provide a proof that a packet receipt does not exist for the packet's sequence by the specified timeout. Of course, timing out a packet on an UNORDERED channel simply triggers the application-specific timeout logic for that packet, and does not close the channel.
 
 For this reason, most modules should use UNORDERED channels as they require less liveness guarantees to function effectively for users of that channel.
 
-### [Acknowledgements](https://github.com/cosmos/ibc-go/blob/main/modules/core/04-channel)
+### [Acknowledgments](https://github.com/cosmos/ibc-go/blob/main/modules/core/04-channel)
 
-Modules may also choose to write application-specific acknowledgements upon processing a packet. This may either be done synchronously on `OnRecvPacket`, if the module processes packets as soon as they are received from IBC module. Or they may be done asynchronously if module processes packets at some later point after receiving the packet.
+Modules can also choose to write application-specific acknowledgments upon processing a packet. This acknowledgment can be done synchronously on `OnRecvPacket` if the module processes packets as soon as they are received from IBC module. Or acknowledgments can be done asynchronously if module processes packets at some later point after receiving the packet.
 
-Regardless, this acknowledgement data is opaque to IBC much like the packet `Data` and will be treated by IBC as a simple byte string `[]byte`. It is incumbent on receiver modules to encode their acknowledgemnet in such a way that the sender module can decode it correctly. This should be decided through version negotiation during the channel handshake.
+Regardless, this acknowledgment data is opaque to IBC much like the packet `Data` and is treated by IBC as a simple byte string `[]byte`. It is incumbent on receiver modules to encode their acknowledgment so that the sender module can decode it correctly. This should be decided through version negotiation during the channel handshake.
```

Comment:
```suggestion
Regardless, this acknowledgment data is opaque to IBC much like the packet `Data` and is treated by IBC as a simple byte string `[]byte`. It is incumbent on receiver modules to encode their acknowledgment so that the sender module can decode it correctly. Define this encoding through version negotiation during the channel handshake.
```

Reply:
I probably guessed incorrectly but again the unclear reference "This" in "This should be decided"

are we talking about the encoding? or ? and how is it decided?

Reply:
I think we're talking about how it is decided the encoding, so that both parties know how to encode/decode to understand each other.

Reply:
`The encoding should be negotiated between the two parties during version negotiation in the channel handshake.

Reply:
since this is the overview topic, I suggest that we are too specific here.

Readers can learn more at https://docs.cosmos.network/master/ibc/custom.html#channel-handshake-version-negotiation 
---
oh "should" is such an ambiguous word, let's avoid per this advice
https://developers.google.com/style/word-list?hl=en#letter-s 

**should, should be
Generally avoid.**
![image](https://user-images.githubusercontent.com/10067215/142517735-4665c117-9deb-46ae-a413-2aede50b63ea.png)